tests/ui/asm/cfg-parse-error.rs

```diff
@@ -0,0 +1,45 @@
+//@ needs-asm-support
+#![feature(asm_cfg)]
+
+use std::arch::asm;
+
+fn main() {
```

Comment:
Could you also add tests for:
- Multiple `#[cfg]` on the same operand.
- Non-cfg attributes being properly rejected.
- Inner attributes are rejected.

Reply:
I've added a test for each of these cases.